views/tasks/active-psc-details.html

```diff
@@ -177,10 +118,10 @@ <h2 class="govuk-heading-m">1 other registrable person</h2>
             <div class="govuk-summary-card govuk-!-margin-bottom-4">
               <div class="govuk-summary-card_heading govuk-!-padding-top-4 govuk-!-padding-bottom-4 govuk-!-padding-right-3 govuk-!-padding-left-3" style="display: flex; justify-content: space-between; align-items: center">
                 <p class="govuk-!-font-weight-bold govuk-!-margin-0">AYR BAPTIST CHURCH</p>
-                 {{govukTag({
-                  text: "Active",
-                  classes: "govuk-tag--green"
-                 })}}
+                {{govukTag({
+                text: "Active",
+                classes: "govuk-tag--green"
```

Comment:
How come you removed the indentation in this tag and the one above it?

Reply:
This code is due to come out in the other subtasks.